src/Compilers/VisualBasic/Portable/Generated/VisualBasic.Grammar.g4

```diff
@@ -0,0 +1,1461 @@
+﻿// <auto-generated />
+grammar vb;
+
+compilation_unit
+  : option_statement* imports_statement* attributes_statement* statement* punctuation
+  ;
+
+option_statement
+  : 'Option' ('Explicit' | 'Strict' | 'Compare' | 'Infer') ('On' | 'Off' | 'Text' | 'Binary')?
+  ;
+
+imports_statement
+  : 'Imports' (imports_clause(',' imports_clause)*)?
+  ;
+
+imports_clause
+  : simple_imports_clause
+  | xml_namespace_imports_clause
+  ;
+
+simple_imports_clause
+  : import_alias_clause? name
+  ;
+
+import_alias_clause
+  : identifier_token '='
+  ;
+
+xml_namespace_imports_clause
+  : '<' xml_attribute '>'
+  ;
+
+xml_attribute
+  : xml_node '=' xml_node
+  ;
+
+attributes_statement
+  : attribute_list*
+  ;
+
+attribute_list
+  : '<' (attribute(',' attribute)*)? '>'
+  ;
+
+attribute
+  : attribute_target? type argument_list?
+  ;
+
+attribute_target
+  : 'Assembly' ':'
+  | 'Module' ':'
+  ;
+
+argument_list
+  : '(' (argument(',' argument)*)? ')'
+  ;
+
+argument
+  : omitted_argument
+  | range_argument
+  | simple_argument
+  ;
+
+omitted_argument
+  : punctuation
```

Comment:
this looks wrong.  not properly picking up "empty token" for some reason.  will investigate.

Reply:
fixed.